frontend/src/models/memory.js

```diff
@@ -17,23 +17,32 @@ export class PercentageMemory {
     this.playedCards = [];
   }
 
-  memorize(playedCard) {
-    if (Math.random() <= this.percent) this.playedCards.push(playedCard);
+  memorize(playedCard, trickId = null) {
+    if (Math.random() <= this.percent)
```

Comment:
I know this is just a formatting change here but thought I'd tell you about this one: There's a `chance(percentage)` helper function in `random.js` that's basically doing just this - the logic here is simple enough so I don't have hard feelings about using that helper. It's just that I personally regularly mess up probability logic and love to have this kind of helper around :)

Reply:
I'm quoting a song: "I couldn't care less" 😉
I usually don't write too much JavaScript, so I didn't know about that one. I'm happy with anything that works and is decent to read 👍🏻